packs/BtRK.json

```diff
@@ -0,0 +1,70 @@
+{
+    "cgdbId": 49,
+    "code": "BtRK",
+    "name": "Beneath the Red Keep",
+    "releaseDate": null,
+    "cards": [
+        {
+            "code": "13062",
+            "type": "location",
+            "name": "Tower of the Hand",
+            "unique": true,
+            "faction": "stark",
+            "loyal": false,
+            "cost": 1,
+            "traits": [
+                "King's Landing"
+            ],
+            "text": "<b>Reaction:</b> After a card you control comes out of shadows, if there is a <i>Winter</i> plot card revealed, choose and stand a [stark] character. (Limit once per phase.)",
+            "deckLimit": 3
+        },
+        {
+            "code": "13064",
+            "type": "location",
+            "name": "The Maidenvault",
+            "unique": true,
+            "faction": "tyrell",
+            "loyal": false,
+            "cost": 2,
+            "traits": [
+                "King's Landing"
+            ],
+            "text": "Shadow (1).\n<b>Reaction:</b> After The Maidenvault comes out of Shadows, choose a participating <i>Lady</i> character. Stand it and remove it from the challenge.\n<b>Interrupt:</b> When the challenges phase ends, kneel the Maidenvault to return it to the shadows.",
```

Comment:
should be:
the card text has a few minor issues. should be:
```
Shadow (1).\n<b>Reaction:</b> After The Maidenvault comes out of shadows, choose a participating <i>Lady</i> character. Stand it and remove it from the challenge.\n<b>Interrupt:</b> When the challenges phase ends, kneel The Maidenvault to return it to shadows.
```

Reply:
Thanks, good catch!